app/models/manageiq/providers/autosde/storage_manager/cloud_volume.rb

```diff
@@ -7,12 +7,10 @@ class ManageIQ::Providers::Autosde::StorageManager::CloudVolume < ::CloudVolume
 
   def self.raw_create_volume(_ext_management_system, _options = {})
     # @type [StorageService]
-    storage_service = _options[:storage_service]
-
     vol_to_create = _ext_management_system.autosde_client.VolumeCreate(
-      :service => storage_service.ems_ref,
-      :name    => _options[:name],
-      :size    => _options[:size]
+      :service => _options["storage_service"],
```

Comment:
A leading underscore indicates the var is unused, if you're actually using`_options` you should rename it `options`